gradio/components.py

```diff
@@ -952,6 +956,7 @@ def __init__(
             self.type = type
 
         self.type = type
+        self.output_type = "auto"
```

Comment:
Do we need this? Can we just remove this line (see my comment below as well)

Reply:
I actually wanted to keep it that way as we may want to have changes in the future 
- where we could not support `auto` type in some components
- want to enable users to declare a specific output_type